03_operators.ipynb

```diff
@@ -30,10 +30,10 @@
     {
      "data": {
       "text/html": [
-       "<pre class=\"language-APL\">┌→─────────────────────────────────────┐\n",
+       "<span style=\"white-space:pre; font-family: monospace\">┌→─────────────────────────────────────┐\n",
```

Comment:
This line means that you've got an old version of the APL notebook kernel. It breaks the formatting. Could you please update your kernel and then re-run all the cells?

Reply:
@jph00  - Should be fixed now.